etc/databases/world/updates/updates.sql

```diff
@@ -14466,6 +14466,13 @@ begin not atomic
 
         insert into applied_updates values ('101120221');
     end if;
+
+    -- 24/11/2022 1
+    -- Crippling Poison charges.
+    if (select count(*) from applied_updates where id='241120221') = 0 then	
+	    INSERT INTO `alpha_world`.`spell_enchant_charges` (`entry`, `charges`) VALUES ('3408', '1');		
+	    insert into applied_updates values ('241120221');
+    end if;
```

Comment:
Existing entries in `spell_enchant_charges` should be removed (2823, 2824, 5761). These poisons don't have charges in alpha.

```
+-------+---------+---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------+
| entry | charges | name_enus           | description_enus                                                                                                                                            |
+-------+---------+---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------+
|  2823 |      60 | Poison Damage       | Coats a sword or dagger with poison that lasts for 30 minutes.
Each strike has a chance of poisoning the enemy for 32 damage over 12 seconds.               |
|  2824 |      75 | Poison Damage II    | Coats a sword or dagger with poison that lasts for 30 minutes.
Each strike has a chance of poisoning the enemy for 48 damage over 12 seconds.               |
|  5761 |      50 | Mind-numbing Poison | Coats a sword or dagger with poison that lasts for 30 minutes.
Each strike has a chance of poisoning the enemy, doubling their casting time for 15 seconds. |
+-------+---------+---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------+